.github/workflows/ci.yml

```diff
@@ -0,0 +1,26 @@
+---
+name: Docker Image CI
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
```

Comment:
shouldn't we build on centos?

Reply:
This is already the case, this image is only used to run the docker image based on centos.